services/pgnotifier/pgnotifier.go

```diff
@@ -203,7 +203,7 @@ func (notifier *PgNotifierT) trackBatch(batchID string, ch *chan []ResponseT) {
 						Error:  jobError.String,
 					})
 				}
-				rows.Close()
+				_ = rows.Close()
```

Comment:
log as warn ?

Reply:
Same as https://github.com/rudderlabs/rudder-server/pull/2064#discussion_r899900377
Let's agree on one then I'll review this one too.